pkg/workloads/cortex/serve/init/bootloader.sh

```diff
@@ -118,6 +119,37 @@ create_s6_service() {
     chmod +x $dest_script
 }
 
+# terminate pod if this process exits (zero or non-zero exit code)
+create_s6_task() {
+    # good pages to read about s6-overlay
```

Comment:
Should we move these comments at the top of this script and mention that it's for the `create_s6_service` and `create_s6_task` functions?